static/js/message_events.js

```diff
@@ -302,6 +302,19 @@ exports.update_messages = function update_messages(events) {
 
         notifications.received_messages([msg]);
         alert_words.process_message(msg);
+
+        if (topic_edited || stream_changed) {
+            // if topic is changed
+            let possible_old_topic = util.get_edit_event_orig_topic(event);
+            let possible_new_topic = new_topic;
+
+            if (!topic_edited) {
+                possible_old_topic = msg.topic;
+                possible_new_topic = possible_old_topic;
+            }
```

Comment:
I think these variable names could be just `post_edit_topic` and `pre_edit_topic` without the `possible`?

Reply:
Done!